### PR TITLE
miniassigment w3d1 Dog + Palindrome + FileUtils

### PR DESCRIPTION
**Pull request by diegotc86**
Files to_do_list.rb and to_do_class.rb created
execute to_do_list.rb to run the program

### CODE REVIEW

File: week-1/summer_team/tic_tac_toe.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
+A= [["-","-","-"],["-","-","-"],["-","-","-"]]
```
Comment (DiegoCuevas) [inline week-1/summer_team/tic_tac_toe.rb]: archivo innecesario (?)

**Comment by KattyaCuevas** [comment]
There are a lot of additional files.

File: week-2/to_do_list/main.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require_relative "to_do_list"
+
+system 'clear'
```
Comment (DiegoCuevas) [inline week-2/to_do_list/main.rb:3]: why did you use "system" here?

File: week-2/to_do_list/to_do_class.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    @task_list = []
+  end
+
+  def list
+    #PAULO
+    #Lista elementos de @task_list
+    #puts @task_list.empty? ? "[Lista vacia] \n ¿Te llevas bien con todos?" : "Esta es tu lista de venganza \n =============================="
+    result = []
+    @task_list.each_with_index do |task , id|
+      result << "#{id + 1} - #{task}" if task != ""
```
Comment (dotconde) [inline week-2/to_do_list/to_do_class.rb:12]: Chévere, muy entendible !

File: week-2/to_do_list/to_do_class.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    end
+    result
+  end
+
+  def add(task)
```
Comment (dotconde) [inline week-2/to_do_list/to_do_class.rb:17]: ¿Qué pasa si inserto una cadena de espacios?

Comment (dotconde) [reply week-2/to_do_list/to_do_class.rb:17]: 🗡

File: week-2/to_do_list/to_do_class_rspec.rb [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  to_do = ToDo.new
+  
+  describe "Test Class" do
+
+    it "Add task into to_do" do
+        expect(to_do.add("Read more about El Niño")).to eq(["Read more about El Niño"])
+    end
+
+    it "Add task into to_do" do
+      expect(to_do.add("Write my first blog post!")).to eq(["Read more about El Niño","Write my first blog post!"])
```
Comment (KattyaCuevas) [inline week-2/to_do_list/to_do_class_rspec.rb:14]: I really like this type of tests! 
Very readable! Thanks!! 🎉

File: week-2/to_do_list/to_do_class_rspec.rb [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    end
+  end
+
+  describe "Test Main" do
+
+    it "Operation wrong input" do
+        expect(operation("jdfnhdjkfv", to_do)).to eq("Error: Unexpected input") 
+    end
+
+  end
```
Comment (dotconde) [inline week-2/to_do_list/to_do_class_rspec.rb:41]: All tests worked for me, great !

File: week-2/to_do_list/to_do_list.rb [hunk 7]
```diff
@@ -0,0 +1,48 @@
+require_relative "to_do_class"
+
+def operation(user_input, task_list)
+  #LIAN y DIEGO
+  #Ejecuta acción según el input del usuario
+
+  if !!(user_input =~ /^t$/)
```
Comment (KattyaCuevas) [inline week-2/to_do_list/to_do_list.rb:7]: This is a little bit confused. Maybe you can use the `scan` method ? 
Something like user_input.scan(/^t$/).any?

File: week-2/to_do_list/to_do_list.rb [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+def instructions
+  puts "***************************************************************"
+  puts 'Bienvenido al gran To Do List "Venganza"'
+  puts "***************************************************************"
+  puts "Para listar tus tareas digita: t"
+  puts "Para agregar una nueva tarea digita: t [tarea]"
+  puts "Para eliminar una tarea digita: t -d [numero de tarea]"
+  puts "Para salir del To Do List digita: q"
+  puts "***************************************************************"
```
Comment (KattyaCuevas) [inline week-2/to_do_list/to_do_list.rb:38]: Thanks for the instructions!

**Pull request closed without merging**